### ISSUE TITLE
Transparency rendering issue with stick_ball representation

### ISSUE THREAD

**Issue by giagitom** [schrodinger/pymol-open-source#146]
Hi,
This [pse](https://www.dropbox.com/s/5do6ylbcragiphi/issue_stick_ball_orthoscopic_transparency.pse?dl=0) should explain the issue, just try to render it (older versions of pymol have the same issue). 

Basically, as you can see from the rendered image, transparent material seems to be opaque for some atoms in a particular situation. 
This happens only when ball_stick and orthoscopic are enabled and only on atoms with 3 or more connections! 

Rendering vs screen
![image](https://user-images.githubusercontent.com/20627977/98752920-78f74d00-23c3-11eb-99bd-40895fa8e4c6.png)
![image](https://user-images.githubusercontent.com/20627977/98752948-84e30f00-23c3-11eb-8bb8-b9ca5d1aad67.png)

**Comment by speleo3** [comment]
I was about to suggest `transparency_mode` 1 or 3 (multi-layer), but I hit a different (maybe related) bug, some spheres become fully opaque:
![opaque-spheres-bug](https://user-images.githubusercontent.com/1239490/98797935-8c2f0a80-240d-11eb-8af8-90ae4dca33d6.png)

**Comment by giagitom** [comment]
Exactly.. it's something really weird related also to the field of view I think

**Comment by speleo3** [comment]
Can reproduce in PyMOL 1.8.2 but not in 1.8.0. However, there are almost no changes to the ray tracer or the spheres representation between those versions :thinking:

**Comment by giagitom** [comment]
Maybe is something related to ```stick_ball``` setting? 
The transparency issue happens only with sticks representation with ```stick_ball, on```. In the provided pse transparency is applied on sticks representation and spheres are small and fully opaque
